packages/e2e-contract-tests/.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
I think we don't need another lint config on the project as it's going to use the root one.